changelogs/unreleased/478-update-index-rst.yml

```
@@ -0,0 +1,5 @@
+description: Update index.rst with so that it contains the changes that where do to the same file in inmanta-service-orchestrator
+issue-nr: 478
+issue-repo: inmanta-service-orchestrator
+change-type: patch
+destination-branches: [master, iso7]
```

```suggestion
destination-branches: [master, iso7, iso6]
```

While iso6 does already use this file, I think it should also get these fixes.